website/manual.md

```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
 
 - [A pixelated animation one by @tanakaworld](https://github.com/denoland/deno/blob/master/website/images/deno_logo_4.gif)
 
+## Frequently Asked Questions
+
+### Is Deno production ready?
+
+No.
+
+### Will there be an official package manager for Deno?
+
+No.
```

Comment:
Expand on this or remove.

Reply:
You want to add the example of the issues with `npm` and the fact we don't really know what we download?